engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -203,9 +205,24 @@ def preprocess_requests(self, requests):
 
     def postprocess_results(self):
         results = []
-        for i in range(len(self.active_requests)):
-            req = self.active_requests[i]
-            res = {"data": req.get_next_token(), "last": req.is_last_token()}
+        err_reqs = dict((r.id, err) for r, err in self.error_requests)
```

Comment:
why we need this error requests?

Reply:
In case there is a issue with prefill for one of the requests for the rolling batch. If one of the requests has input length > max_seq_len we return that as an error request instead of failing the whole prefill batch

Reply:
If you belive we should have a global mechanism to handle this behavior, we can prepare another PR and do with all backends in 1 go